core/primitives/version.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "primitives/version.hpp"
+
+#include <algorithm>
+
+#include "crypto/hasher/hasher_impl.hpp"
+
+namespace {
+  static std::unique_ptr<kagome::crypto::Hasher> kHasher =
```

Comment:
Please comment why, as it breaks DI principle we try to follow (I understand why, I just think a comment would be good here).